article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -148,48 +150,42 @@ object ArticlePicker {
-    // when an article is not supportted
-    val forceDCR = request.forceDCR
-    forceDCR || isInWhitelist(request.path)
+  def dcrForced(request: RequestHeader): Boolean = {
+    request.forceDCR || isInWhitelist(request.path)
   }
 
-  def dcrShouldNotRender(request: RequestHeader): Boolean = {
+  def dcrDisabled(request: RequestHeader): Boolean = {
     val forceDCROff = request.forceDCROff
     val dcrEnabled = conf.switches.Switches.DotcomRendering.isSwitchedOn
     forceDCROff || !dcrEnabled
   }
 
   def getTier(page: PageWithStoryPackage)(implicit request: RequestHeader): RenderType = {
     val whitelistFeatures = featureWhitelist(page, request)
-    val userIsInDotcomRenderingCohort = ActiveExperiments.isParticipating(DotcomRendering)
-    val userIsInDiscussionRenderingCohort = ActiveExperiments.isParticipating(DiscussionRendering)
-    val userIsInDCRBubbleCohort = ActiveExperiments.isParticipating(DCRBubble)
-    val additionalChecksForRegularCohort = ArticlePageChecks.isDiscussionDisabled(page) && ArticlePageChecks.isNotOpinion(page)
-
-    // Decide if we should render this request with the DCR platform or not
-    val tier = if (dcrShouldNotRender(request)) {
-      LocalRenderArticle
-    } else if (dcrShouldRender(request) || userIsInDCRBubbleCohort) {
-      RemoteRender
-    } else if (primaryChecksForDCRRendering(page, request) && additionalChecksForRegularCohort && userIsInDotcomRenderingCohort) {
-      RemoteRender
-    } else if (primaryChecksForDCRRendering(page, request) && userIsInDiscussionRenderingCohort) {
-      RemoteRender
-    } else {
-      LocalRenderArticle
-    }
+    val userInMainTest = ActiveExperiments.isParticipating(DotcomRendering)
+    val userInDiscussionTest = ActiveExperiments.isParticipating(DiscussionRendering)
+    val userInDCRBubble = ActiveExperiments.isParticipating(DCRBubble)
+
+    val canRender = dcrCanRender(page, request)
+
+    val tier =
+      if (dcrDisabled(request)) LocalRenderArticle
+      else if (dcrForced(request)) RemoteRender
+      else if (userInDCRBubble) RemoteRender
+      else if (userInMainTest && canRender && notCommentOrOpinion(page)) RemoteRender
+      else if (userInDiscussionTest && canRender) RemoteRender
+      else LocalRenderArticle
 
     val isArticle100PercentPage = dcrArticle100PercentPage(page, request);
     val isAddFree = ArticlePageChecks.isAdFree(page, request)
 
     // include features that we wish to log but not whitelist against
     val features = whitelistFeatures +
-      ("userIsInCohort" -> userIsInDotcomRenderingCohort) +
-      ("userIsInCohortDiscussion" -> userIsInDiscussionRenderingCohort) +
+      ("userIsInCohort" -> userInMainTest) +
+      ("userIsInCohortDiscussion" -> userInDiscussionTest) +
       ("isAdFree" -> isAddFree) +
-      ("isArticle100PercentPage" -> isArticle100PercentPage)
+      ("isArticle100PercentPage" -> isArticle100PercentPage) +
+      ("dcrCouldRender" -> canRender)
```

Comment:
This line is the part that adds logging. So we will see `dcrCouldRender` as a field in Kibana.